wgsl/index.bs

```diff
@@ -3468,8 +3498,12 @@ Different call sites may supply pointers into different originating variables.
 
 ### Invalid Memory Reference ### {#invalid-mem-reference}
 
-If a reference or pointer access is out of bounds, an
-<dfn noexport>invalid memory reference</dfn> is produced.
+A reference or a pointer that accesses one or more [=memory locations=] outside
```

Comment:
The operation is the one doing the access, not the reference or pointer. this is a problem with the original text.  How about:

> An operation that accesses one or more memory locations outside those associated with the operation' view is an out of bounds access.

?